FreeRTOS/FreeRTOS
# Update submodule pointers to their respective head revisions.

Description
-----------
Run "git submodule update --remote --merge" to update all the submodules to their respective head revisions.

Test Steps
-----------
Just checked the Win32 demo (which runs most of the integration tests) still executed with no errors.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
